skills/eval-run/SKILL.md

```
@@ -0,0 +1,153 @@
+- `--no-judge` (skip LLM judges)
+- `--gold` (save outputs as gold references after run)
+
+Check if eval.yaml exists:
+
+```bash
+test -f eval.yaml && echo "CONFIG_EXISTS" || echo "NO_CONFIG"
+```
+
+**If eval.yaml is missing**: invoke eval-analyze to bootstrap:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)